packages/workflow/docs/Workflow.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { Button } from "@kaizen/button"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { Workflow, WorkflowExit, WorkflowProps } from "../"
+
+const defaultArgs = {
+  workflowName: "Create a self-reflection cycle",
+  stepName: "Settings",
+  steps: ["Settings", "Questions", "Preview", "Employees", "Schedule"],
+  isComplete: false,
+  status: {
+    content: "Draft",
+    vairant: "statusDraft",
+  },
+  previousAction: <Button reversed label="back" />,
+  nextAction: <Button reversed label="next" />,
```

Comment:
These don't look very good in the controls, try something like the `validationState` in [FilterSelect](https://dev.cultureamp.design/SR2-73/add-workflow-components/?path=/docs/components-filter-select--docs) where people can toggle between `undefined` and a predetermined component.